test/features/sdk_flow.feature

```diff
@@ -206,19 +206,19 @@ Feature: SDK File Upload Tests
 #   Until monster is updated to support launching Chrome with arguments (--use-fake-ui-for-media-stream, --use-fake-device-for-media-stream)
 #   this full test will fail in Travis
 #
-    Scenario Outline: I should be able to see a permission priming screen before trying to capture using my webcam.
-      Given I initiate the verification process with <locale>
-      And I do have a camera
-      Then I should see 3 document_select_buttons ()
-      When I click on passport ()
-      Then I can confirm privacy terms
-#      Then I see the camera permissions priming screen
-#      Then page_title should include translation for "capture.passport.front.title"
-
-      Examples:
-        | type | locale |
-        |      |        |
-        | pdf  | es     |
+#     Scenario Outline: I should be able to see a permission priming screen before trying to capture using my webcam.
+#       Given I initiate the verification process with <locale>
+#       And I do have a camera
+#       Then I should see 3 document_select_buttons ()
+#       When I click on passport ()
+#       Then I can confirm privacy terms
+# #      Then I see the camera permissions priming screen
+# #      Then page_title should include translation for "capture.passport.front.title"
+
+#       Examples:
+#         | type | locale |
+#         |      |        |
+#         | pdf  | es     |
```

Comment:
Is there a reason for keeping this Cucumber test around as commented out code?

Reply:
In case the Legal team will ask us to turn on this feature (currently disabled behind feature flag), we don't have to write it again.

Also, ideally we should have this covered with JS UI test but, I'm I have never seen these screen but probably it's nothing complicated to add the test for. Maybe worth removing it for now completely and once we will introduce this feature again, we will add the JS UI test? 

What do you think @kopijunkie and also @stefaniacardenas ? 

Btw. I added a comment to this scenario why it's commented out for now.

Reply:
I kind of feel like the file's git history should be a good enough record of this. Or just add the relevant test from scratch when the time comes since it is highly likely to be a JS UI test by then.

Reply:
Yeah, I'd be up for deleting that completely and adding JS UI tests when we activate this feature again. Just waiting for @stefaniacardenas's opinion on that too :)

Reply:
Yes, I agree with deleting it

Reply:
Done in:
c27b602